lib/modules/manager/asdf/extract.spec.ts

```diff
@@ -184,6 +206,13 @@ dummy 1.2.3
             datasource: 'flutter-version',
             depName: 'flutter',
           },
+          {
+            currentValue: '0.41.2',
+            datasource: 'github-tags',
+            packageName: 'fluxcd/flux2',
+            depName: 'flux2',
+            extractVersion: '^v(?<version>\\S+)',
```

Comment:
```suggestion
            extractVersion: '^v(?<version>.+)',
```
i don't think git tags can have spaces?

Reply:
I have no problem changing it, though I did copy what most (all?) of the other blocks were using for `extractVersion`.

We could
- Change just this one
- Change all of them
- Leave it as `\\S`

I'm happy do whichever one is requested 😄

Reply:
Change all new, we can change the existing in a seperate PR.